packages/storybook/src/templates/componenten/index.tsx

```diff
@@ -46,6 +47,16 @@ export default function Componenten() {
     [debouncedSearchTerm, selectedFrameworks],
   );
 
+  const frameworkCounts = useMemo(() => {
+    const counts: { [key: string]: number } = {};
+
+    frameworkOptions.forEach((framework) => {
+      counts[framework] = allComponentsData.filter((component) => component.frameworks.includes(framework)).length;
+    });
+
+    return counts;
+  }, []);
+
```

Comment:
De `forEach` kan beter een `reduce` zijn, omdat dit het patroon volgt van een array omzetten naar 1 waarde.

```suggestion
  const frameworkCounts: { [key: string]: number } = useMemo(() =>
    frameworkOptions.reduce((counts, framework) => {
      counts[framework] = allComponentsData.filter((component) => component.frameworks.includes(framework)).length;
    }),
  [frameworkOptions.forEach]);

```

Reply:
✅